applications/h264/grpc_h264_endoscopy_tool_tracking/cpp/metadata.json

```diff
@@ -0,0 +1,59 @@
+{
+	"application": {
+		"name": "gRPC-streaming H.264 Endoscopy Tool Tracking Distributed",
+		"authors": [
+			{
+				"name": "Holoscan Team",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"language": "C++",
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"dockerfile": "applications/h264/Dockerfile",
+		"holoscan_sdk": {
+			"minimum_required_version": "2.6.0",
+			"tested_versions": [
+				"2.6.0"
```

Comment:
Can you confirm whether this is tested with the latest HoloHub, or at least the HoloHub `holoscan-sdk-2.6.0` tag? Want to confirm with respect to the recent Endoscopy Tool Tracking optimizations @AndreasHeumann  has already commented on

Reply:
Yes, I rebased the latest Holohub and tested with 2.6.